smac/scenario/scenario.py

```diff
@@ -77,6 +77,10 @@ def __init__(self, scenario, cmd_args=None, run_id=1):
         self._groups = defaultdict(set)
         self._add_arguments()
 
+        # Make cutoff mandatory if run_obj is runtime
+        if 'run_obj' in scenario and scenario['run_obj'] == 'runtime':
```

Comment:
Not sure where that would be? To not have it in scenario, a lot of code would need changes I think.